{{cookiecutter.project_slug}}/Makefile

```diff
@@ -28,11 +28,14 @@ compile-translations: ## Compile .po files with ./manage.py compilemessages
 	./manage.py compilemessages
 
 format:  # Fix some linting issues in the project
-	black {{ cookiecutter.project_slug }} fabfile.py
-	isort {{ cookiecutter.project_slug }} fabfile.py
+	ruff format {{ cookiecutter.project_slug }} fabfile.py
+	ruff check --select I --fix {{ cookiecutter.project_slug }} fabfile.py
```

Comment:
Why do we specify only isort rules here? Shouldn’t we fix everything we can? Or is it the only thing it can fix and it’s a performance optimisation?